### PR TITLE
Fixes #290 - Flight information Display

### LINKED ISSUE #290: AFIS/AG profile (Flight Information Display - FID)
## Is your feature request related to a problem? Please describe.
There is currently no profile tailored specifically for providing an A/G or AFIS service.

## Describe the solution you'd like
Add profile(s) with:
- Custom lists containing the most useful information
- Custom Voice.txt and Profiles.txt files to include all 'approved' positions (for discussion with @cbyworth)
- Custom symbology etc.
- Flight Information Display (FID)
- Ground View

## Is your suggestion based on a real world display?
[SimpliFID](http://www.simplifid.aero/)

[Barton Trial](https://www.suasnews.com/2019/09/airspace4all-ga-airfield-ats-ads-b-traffic-display-trial-at-barton-to-continue-until-30-april-2020/)

## Additional context
See [CAP 670 SA 2021/02](https://publicapps.caa.co.uk/docs/33/CAP670%20SA%202021-02.pdf) - Cooperative Surveillance Systems & Flight Information Displays

### PR DESCRIPTION
**Pull request by SamLefevre**
Fixes #290 

# Summary of changes

Add FID to simulate https://databeacon.aero/simplifid/

# Screenshots (if necessary)

image

### CODE REVIEW

**Comment by SamLefevre** [comment]
I'll finish the rest of the voice.txt by the end of the week

**Comment by luke11brown** [comment]
If simulating an ADS-B derived display, would it be an idea to get it to use S-Mode correlation in general settings?

**Review by kye-taylor — CHANGES_REQUESTED** [review]
I think we could include London and Scottish Infomation positions here also.

Looks really clean and great, As for the ATIS information, I'm not sure what best to settle with so I have to guess a little. Feel free to change it!

File: UK/Data/Settings/Voice_FID.txt
Comment (luke11brown) [inline UK/Data/Settings/Voice_FID.txt:1]: Can we also include all the positions listed here?

https://www.vatsim.uk/atc/endorsements#endorsement-afis

**Review by kye-taylor — APPROVED** [review]

**Review by luke11brown — APPROVED** [review]

**Pull request merged**